spring-batch-bigquery/pom.xml

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright 2006-2021 the original author or authors.
```

Comment:
2006 should be removed here (I've also seen 2002 in other files).

Reply:
Done